src/zkevm_specs/evm/execution/__init__.py

```diff
@@ -110,6 +111,8 @@
     ExecutionState.STOP: stop,
     ExecutionState.RETURN: return_revert,
     ExecutionState.ErrorInvalidJump: invalid_jump,
```

Comment:
renaming `xxx.py` to `error_xxx.py` is good 👍  

How about also rename its function name to achieve ultimate consistent